sql/create/release-1.0.0_schema/mysql/escheduler_ddl.sql

```diff
@@ -176,8 +176,8 @@ CREATE TABLE `t_escheduler_project` (
   `desc` varchar(200) DEFAULT NULL COMMENT '项目描述',
   `user_id` int(11) DEFAULT NULL COMMENT '所属用户',
   `flag` tinyint(4) DEFAULT '1' COMMENT '是否可用  1 可用,0 不可用',
-  `create_time` datetime DEFAULT CURRENT_TIMESTAMP COMMENT '创建时间',
```

Comment:
if `create_time` field doesn't have default value,  when create project, can save to database normally?  I'm not sure for this

if could add unit test will be better